content/docs/started/getting-started-docker-desktop.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Getting Started with Kubeflow on Docker Desktop"
+weight = 10
++++
+
+This document outlines the steps to get Kubeflow running with Docker Desktop. Docker Desktop allows you to build production ready applications locally. 
+
+By the end of this tutorial, you should be able to get Kubeflow running and access the Kubeflow Dashboard.
+
+Pre-requisites:
+
+- Laptop, Desktop or a Workstation
+- Mac OS X or Windows 10
+- Docker Desktop (https://www.docker.com/products/docker-desktop) for Mac or Windows
+- sudo or admin access on the local machine
+- Access to an Internet connection with reasonable bandwidth
+
+# Enable Kubernetes 
+
+1. Click on the Docker Desktop logo and select 'Preferences...' 
+2. On the 'Kubernetes' tab tick the 'Enable Kubernetes' checkbox and then click on 'Apply'. 
+
+# Install kubectl
+
+```bash
+$ brew install kubectl
+```
+
+# Download and install Kubeflow 
+
+```bash
+KUBEFLOW_SRC=${PWD}/kubeflow
+export KUBEFLOW_TAG=v0.5.1
+
+mkdir ${KUBEFLOW_SRC}
+cd ${KUBEFLOW_SRC}
+
+curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_TAG}/scripts/download.sh | bash
+```
+
+**KUBEFLOW_SRC** - directory where you want kubeflow source to be downloaded
+
+**KUBEFLOW_TAG** - a tag corresponding to the version to checkout such as v0.5.1
+
+**KFAPP** - the name of a directory where you want kubeflow configurations to be stored. 
+
+# Install Kubeflow
+
+Run the following to get kubeflow working: 
+
+```bash
+KFAPP=kfapp
+KUBEFLOW_REPO=${KUBEFLOW_SRC} ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform docker-for-desktop
+
+cd ${KFAPP}
+${KUBEFLOW_SRC}/scripts/kfctl.sh generate platform docker-for-desktop
+
+mkdir ks_app #in the kf_app directory. 
+cd to ks_app
+${KUBEFLOW_SRC}/scripts/kfctl.sh apply all
```

Comment:
Does this work for you? I get an error saying `env.sh: No such file or directory`

Reply:
It works for me, that env.sh is in the kfapp directory.

Reply:
I will do some additional testing as well today on a different machine.

Reply:
hmm, yeah if I move into the kfapp directory instead of ks_app I get a different error message, but now I get `ERROR finding app root from starting path: : unable to find ksonnet project`

Reply:
I get this error too, though it still worked for me to get kubeflow running. I think this ksonnet project error is strange as I do have ksonnet and it seems to check for the right version etc when  you run the ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform docker-for-desktop line

Reply:
I already posted a longer remark below. I needed to run `ks init ks_app` instead of `mkdir` to get rid of the error and proceed.

Reply:
Thanks @ionicsolutions, I was having issues with that and couldn't work out how to fix it. I will test again and update the tutorial.

Reply:
I tested this today and it works so will update. However, I seem to encounter an issue where I am stuck in a loop of:

'Getting kubeflow namespace ready...[OK]
Bringing kubeflow services up.No resources found.
No resources found.'